test/install_test.go

```diff
@@ -18,9 +18,9 @@ type deploySpec struct {
 	containers []string
 }
 
-//////////////////////
-///   TEST SETUP   ///
-//////////////////////
+// ////////////////////
```

Comment:
intentional?

Reply:
TIOLI: I _much_ prefer simplifying this so it's less brittle.

Something like
```
//
// Test Setup
//
```

Should create just as much visual spacing, but is easier to change, reformatting, etc.

Reply:
Definitely not intentional. It must have been my editor being overly helpful. I'll revert this.